packages/clay-tooltip/src/TooltipProvider.tsx

```diff
@@ -137,10 +137,8 @@ const TooltipProvider = ({
 			isHovered.current = false;
 		}, []),
 		onHide: useCallback(() => {
-			if (!isHovered.current && !isFocused.current) {
```

Comment:
Hey @matuzalemsteles, the reason I removed this was that the `Esc` event wasn't going to work because on of the `isHovered` and `isFocused` is set to `true` whenever the tooltip is displayed.